_data/simple-icons.json

```diff
@@ -10279,6 +10279,11 @@
             "hex": "EC1C24",
             "source": "https://en.wikipedia.org/wiki/File:Shanghai_Metro_Full_Logo.svg"
         },
+        {
+            "title": "ShareX",
+            "hex": "2885F1",
```

Comment:
What made you pick this particular hex out of all available options?

Reply:
It's the dominant colour in the logo according to [Color Thief](https://github.com/lokesh/color-thief) .

Reply:
Happy to go with it, just wasn't sure as there seems to be an equal spread of all 4 in the logo!